core/commands/storage.go

```diff
@@ -328,7 +328,11 @@ Use status command to check for completion:
 				return err
 			}
 		}
-		testFlag := req.Options[testOnlyOptionName].(bool)
+		// set to false if not specified
+		testFlag := false
+		if req.Options[testOnlyOptionName] != nil {
+			testFlag = req.Options[testOnlyOptionName].(bool)
+		}
```

Comment:
there is no need for this logic, by default it's false if not specified. anyway.